content/news/026/index.md

```diff
@@ -67,6 +67,30 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### [BITGUN][bitgun-steam]
+
+[![Gameplay screenshot with lots of zombies and a zombie boss](bitgun.jpeg)][bitgun-steam]
+
+BITGUN ([Steam][bitgun-steam], [Twitter][bitgun-twitter],
+[Discord][bitgun-discord]) by [@LogLogGames][bitgun-twitter] is an action
+roguelike zombie shooter with lots of blood and guns, similar to games like
+Hotline Miami, Nuclear Throne and Heat Signature. The game is built using Godot
+and Rust (via [godot-rust][bitgun-godot-rust]).
+
+We recently re-worked our in-game UI using [egui][bitgun-egui] with
```

Comment:
> We

Could you convert the section to a third-person perspective, please?

Reply:
done